provider/ansible/gcp_utils.py

```diff
@@ -156,7 +156,12 @@ def _credentials(self):
             path = os.path.realpath(os.path.expanduser(self.module.params['service_account_file']))
             return service_account.Credentials.from_service_account_file(path).with_scopes(self.module.params['scopes'])
         elif cred_type == 'serviceaccount' and self.module.params.get('service_account_contents'):
-            cred = json.loads(self.module.params.get('service_account_contents'))
+            try:
+                cred = json.loads(self.module.params.get('service_account_contents'))
+            except json.decoder.JSONDecodeError as e:
+                self.module.fail_json(
```

Comment:
Could this ever end up logging some (or all) of the contents of the JSON creds? (I'm assuming that what we're decoding here contains a private key, which would be bad to have show up in a log file...)

Reply:
That's a great point. It'll make debugging harder, but I don't want creds to be leaked into logs.